testing/test_collection.py

```diff
@@ -127,16 +127,16 @@ def test_foo():
 
 class TestCollectFS:
     def test_ignored_certain_directories(self, pytester: Pytester) -> None:
-        tmpdir = pytester.path
-        ensure_file(tmpdir / "build" / "test_notfound.py")
-        ensure_file(tmpdir / "dist" / "test_notfound.py")
-        ensure_file(tmpdir / "_darcs" / "test_notfound.py")
-        ensure_file(tmpdir / "CVS" / "test_notfound.py")
-        ensure_file(tmpdir / "{arch}" / "test_notfound.py")
-        ensure_file(tmpdir / ".whatever" / "test_notfound.py")
-        ensure_file(tmpdir / ".bzr" / "test_notfound.py")
-        ensure_file(tmpdir / "normal" / "test_found.py")
-        for x in Path(str(tmpdir)).rglob("test_*.py"):
+        tmp_path = pytester.path
+        ensure_file(tmp_path / "build" / "test_notfound.py")
+        ensure_file(tmp_path / "dist" / "test_notfound.py")
+        ensure_file(tmp_path / "_darcs" / "test_notfound.py")
+        ensure_file(tmp_path / "CVS" / "test_notfound.py")
+        ensure_file(tmp_path / "{arch}" / "test_notfound.py")
+        ensure_file(tmp_path / ".whatever" / "test_notfound.py")
+        ensure_file(tmp_path / ".bzr" / "test_notfound.py")
+        ensure_file(tmp_path / "normal" / "test_found.py")
+        for x in Path(str(tmp_path)).rglob("test_*.py"):
```

Comment:
Can remove the conversion to Path now:

```suggestion
        for x in tmp_path.rglob("test_*.py"):
```